pravega/pravega-benchmark
# Issue 32:  Fix gradle wrapper

**Change log description**
Imported the gradle wrapper setting from https://github.com/pravega/pravega to the pravega benchmark tool.

**Purpose of the change**
Fixes #32. 

**What the code does**
Updated gradle directory with the files from https://github.com/pravega/pravega/tree/master/gradle

**How to verify it**
Run `./gradlew distTar`  downloads all required pravega client dependencies and build should be successful.

Signed-off-by: Keshava Munegowda <keshava.munegowda@dell.com>
